.github/workflows/test.yml

```
@@ -48,20 +48,60 @@ jobs:
       - name: Setup Alpine dependencies
         run: |
           cat /etc/alpine-release
-          apk add git go make bash binutils-gold python3
+          apk add git go make bash dash zsh binutils-gold python3
         shell: alpine.sh --root {0}
 
       - name: Checkout
         uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)